### PR TITLE
React1 Week2 homework

### PR DESCRIPTION
**Pull request by na-talia**

### CODE REVIEW

File: react/react1/week2/todolist/src/components/ToDoList.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    id: 3,
+    description: "Eat breakfast",
+  },
+];
+
+const ToDoList = () => {
+  const [todosState, setTodosState] = useState(todos);
+
+  const addTodo = () => {
+    let uniqueId = Math.random().toString(15).slice(2);
```
Comment (shpomp) [inline react/react1/week2/todolist/src/components/ToDoList.js:23]: Creative way and definitely a very unique ID 😀  👍🏽 
I learned something new!

File: react/react1/week2/todolist/src/components/ToDoList.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            key={todo.id}
+            todo={todo}
+            deleteTodo={() => deleteTodo(todo.id)}
+          />
+        ))
+      : "No items...";
+  return (
+    <div>
+      <button onClick={addTodo}>Add todo</button>
+      <ul style={{ listStyleType: "none" }}>{todoItems}</ul>
```
Comment (shpomp) [inline react/react1/week2/todolist/src/components/ToDoList.js]: Inline styles should be avoided by all means, so this could rather be in the style file 🙏🏽

**Review by shpomp — APPROVED** [review]
Very nice job! Nothing much to pick at! 👏🏽 
Good understanding of the basics, keep up the good work! 👍🏽 ⭐

**Comment by na-talia** [comment]
> Very nice job! Nothing much to pick at! 👏🏽 Good understanding of the basics, keep up the good work! 👍🏽 ⭐

Thank you very much for the review, I will remove the inline style 👍

**Pull request merged**